lib/WebLedgerClient.js

```diff
@@ -5,17 +5,15 @@
 
 const constants = require('./constants');
 const {create} = require('apisauce');
-const https = require('https');
 const WebLedgerClientError = require('./WebLedgerClientError');
 
 class WebLedgerClient {
   /**
    * @param {string} hostname - the hostname of the ledger node.
-   * @param {boolean} [strictSSL=true] - validate the host's SSL certificate.
    * @param [logger] - the logger to use.
-   * @param {Agent} [httpsAgent] - a custom `https.Agent` instance.
+   * @param {Agent} [httpsAgent] - a NodeJS HTTPS Agent (`https.Agent`) instance
```

Comment:
Is there jsdoc magic that lets you set the type to `{https.Agent}` or similar that links to node docs?